x-pack/plugins/data_visualizer/public/application/file_data_visualizer/components/about_panel/welcome_content.tsx

```diff
@@ -28,7 +35,16 @@ export const WelcomeContent: FC<Props> = ({ hasPermissionToImport }) => {
   return (
     <EuiFlexGroup gutterSize="xl" alignItems="center">
       <EuiFlexItem grow={false}>
-        <EuiIcon size="xxl" type="addDataApp" className="file-datavisualizer-about-panel__icon" />
+        <EuiIcon
+          size="xxl"
+          type="addDataApp"
+          css={{
+            width: '96px',
+            height: '96px',
+            marginLeft: euiThemeVars.euiSizeXL,
+            marginRight: euiThemeVars.euiSizeL,
+          }}
```

Comment:
Should this be reformatted for Emotion? Something like:

```suggestion
          css={css`
            width: 96px;
            height: 96px;
            ${logicalCSS('margin-left', euiThemeVars.euiSizeXL)}
            ${logicalCSS('margin-right', euiThemeVars.euiSizeL)}
          `}
```

Reply:
Thanks for the review. I'm not sure I understand the advantage of using a template literal over an object for these styles?
Looking at the generated css, it's basically the same.

Reply:
Yeah, technically it'll result in the same outcome. I was just thinking in terms of consistency in the writing of styles. That said, it's not a deal-breaker by any means. If ya'll prefer to write your styles in a mix of formats, I'll defer to you.

Reply:
I agree consistency is better. I've switched away from using template literals in this whole PR in favour of plain objects, which read better to me.
Updated in https://github.com/elastic/kibana/pull/178246/commits/584db08131787fffb3f136c9faeed00e78366a49